### DISCUSSION TITLE
Sendgrid inbound email not working "route not found"

### DISCUSSION THREAD

**Discussion by mstroeve** [chatwoot/chatwoot#4035] [Self hosted]
Environment:
Docker setup with s3 minio 
Caddy frontend server (proxy and ssl)

Already checked:
- works fine with website chat channels
- cloud storage (minio) enabled and works with attachments in website chats (for agents and clients)
- sendgrid setup finished and confimed working. Request is processed but route not found.

This route is mentioned in de setup guide. Any idea what i'am doing wrong ?

Error message:
```
[ddb0fb7c-35dd-4d57-95be-aa10fb093c8c] Started POST "/rails/action_mailbox/sendgrid/inbound_email" for 167.89.117.74 at 2022-02-21 22:16:37 +0000
rails_1     | F, [2022-02-21T22:16:37.686095 #1] FATAL -- : [ddb0fb7c-35dd-4d57-95be-aa10fb093c8c]
rails_1     | [ddb0fb7c-35dd-4d57-95be-aa10fb093c8c] ActionController::RoutingError (No route matches [POST] "/rails/action_mailbox/sendgrid/inbound_email"):
rails_1     | [ddb0fb7c-35dd-4d57-95be-aa10fb093c8c]
```

**Comment by sojan-official** [comment]
@mstroeve have you completed the conversation continuity set up as described in 
https://www.chatwoot.com/docs/self-hosted/configuration/features/email-channel/conversation-continuity

are the required environments variables available for both your web and worker images?
